flake.nix

```diff
@@ -37,1605 +27,114 @@
-                  } ${pkgs.nodejs}/bin/node ${lease-period-prolongator}/dist/index.js")
-                  parachainIds)}
-              '';
-            };
-
-          frontend-static = mkFrontendStatic {
-            subsquidEndpoint = "http://localhost:4350/graphql";
-            picassoEndpoint = "ws://localhost:9988";
-            kusamaEndpoint = "ws://localhost:9944";
-            karuraEndpoint = "ws://localhost:9999";
-          };
-
-          frontend-static-persistent = mkFrontendStatic {
-            subsquidEndpoint =
-              "https://persistent.devnets.composablefinance.ninja/subsquid/graphql";
-            picassoEndpoint =
-              "wss://persistent.devnets.composablefinance.ninja/chain/dali";
-            kusamaEndpoint =
-              "wss://persistent.devnets.composablefinance.ninja/chain/rococo";
-            karuraEndpoint =
-              "wss://persistent.devnets.composablefinance.ninja/chain/karura";
-          };
-
-          frontend-static-picasso-persistent = mkFrontendStatic {
-            subsquidEndpoint =
-              "https://persistent.picasso.devnets.composablefinance.ninja/subsquid/graphql";
-            picassoEndpoint =
-              "wss://persistent.picasso.devnets.composablefinance.ninja/chain/picasso";
-            kusamaEndpoint =
-              "wss://persistent.picasso.devnets.composablefinance.ninja/chain/rococo";
-            karuraEndpoint =
-              "wss://persistent.picasso.devnets.composablefinance.ninja/chain/karura";
-          };
+        };
+        packages = {
+          default = self'.packages.devnet-dali;
 
-          frontend-static-firebase = mkFrontendStatic {
-            subsquidEndpoint =
-              "https://persistent.devnets.composablefinance.ninja/subsquid/graphql";
-            picassoEndpoint =
-              "wss://persistent.devnets.composablefinance.ninja/chain/dali";
-            kusamaEndpoint =
-              "wss://persistent.devnets.composablefinance.ninja/chain/rococo";
-            karuraEndpoint =
-              "wss://persistent.devnets.composablefinance.ninja/chain/karura";
+          subxt = pkgs.callPackage ./code/utils/composable-subxt/subxt.nix { };
+          junod = pkgs.callPackage ./code/xcvm/cosmos/junod.nix { };
+          gex = pkgs.callPackage ./code/xcvm/cosmos/gex.nix { };
+          wasmswap = pkgs.callPackage ./code/xcvm/cosmos/wasmswap.nix {
+            crane = crane.nightly;
           };
 
-          frontend-pablo-server = let PORT = 8002;
-          in pkgs.writeShellApplication {
-            name = "frontend-pablo-server";
-            runtimeInputs = [ pkgs.miniserve ];
-            text = ''
-              miniserve -p ${
-                builtins.toString PORT
-              } --spa --index index.html ${frontend-static}/pablo
-            '';
+          # NOTE: crane can't be used because of how it vendors deps, which is incompatible with some packages in polkadot, an issue must be raised to the repo
```

Comment:
```
          # NOTE: crane can't be used because of how it vendors deps, which is incompatible with some packages in polkadot, an issue must be raised to the repo
```
Do we have a ClickUp ticket to track this?

Reply:
it is polkadot issue afaik, not us.

Reply:
My question was more about raising the issue on their side and keeping track of their progress. 😅

Reply:
that is hard one. its like ask them to use nix afaik. or change their build process with some expanation about nix when they do not use it. thar is hars imho.

Reply:
Yeah I agree with @dzmitry-lahoda, we can inform them about it, but they will most likely note care